tools/pipelines/templates/build-npm-client-package.yml

```diff
@@ -0,0 +1,898 @@
+                inputs:
+                  sourceFolder: '${{ parameters.buildDirectory }}/nyc/report'
+                  targetFolder: $(Build.ArtifactStagingDirectory)/codeCoverageAnalysis
+              - task: Bash@3
+                displayName: Report Code Coverage Comparison
+                condition: and(succeededOrFailed(), eq('${{ parameters.reportCodeCoverageComparison }}', true), eq(variables['ReportDirExists'], 'true'), eq(variables['System.PullRequest.TargetBranch'], 'main'))
+                continueOnError: false
+                env:
+                  ADO_API_TOKEN: '$(System.AccessToken)'
+                  GITHUB_API_TOKEN: '$(githubPublicRepoSecret)'
+                  TARGET_BRANCH_NAME: '$(targetBranchName)'
+                  ADO_BUILD_ID: '$(Build.BuildId)'
+                  GITHUB_PR_NUMBER: '$(System.PullRequest.PullRequestNumber)'
+                  GITHUB_REPOSITORY_NAME: '$(Build.Repository.Name)'
+                  ADO_CI_BUILD_DEFINITION_ID_BASELINE: 48
+                  ADO_CI_BUILD_DEFINITION_ID_PR: 11
+                inputs:
+                  targetType: inline
+                  workingDirectory: '${{ parameters.buildDirectory }}'
+                  script: |
+                    set -eu -o pipefail
+                    echo "Github Repository Name: $GITHUB_REPOSITORY_NAME"
+                    echo "Github PR number: $GITHUB_PR_NUMBER"
+                    echo "ADO Build Number: $ADO_BUILD_ID"
+                    echo "Target Branch Name: $TARGET_BRANCH_NAME"
+                    echo "ADO CI BUILD_DEFINITION_ID for baseline: $ADO_CI_BUILD_DEFINITION_ID_BASELINE"
+                    echo "ADO CI BUILD_DEFINITION_ID for PR: $ADO_CI_BUILD_DEFINITION_ID_PR"
+                    echo "Running code coverage comparison"
+                    flub report codeCoverage --verbose
+
+            # Process test result, include publishing and logging
+              - template: /tools/pipelines/templates/include-process-test-results.yml@self
+                parameters:
+                  buildDirectory: '${{ parameters.buildDirectory }}'
+                  testResultDirs: '${{ parameters.testResultDirs }}'
+
+            templateContext:
+              outputParentDirectory: $(Build.ArtifactStagingDirectory)/codeCoverageAnalysis
+              outputs:
+                - ${{ if eq(parameters.testCoverage, true) }}:
+                      - output: pipelineArtifact
+                        displayName: Publish Artifacts - code-coverage
+                        condition: and( succeededOrFailed(), eq(variables['ReportDirExists'], 'true'))
+                        targetPath: $(Build.ArtifactStagingDirectory)/codeCoverageAnalysis
+                        artifactName: 'codeCoverageAnalysis-$(System.JobAttempt)'
+                        sbomEnabled: false
+                        publishLocation: pipeline
+
+          # Job - Tests
+          - ${{ each test in parameters.taskTest }}:
```

Comment:
Same as previous job but not running anything related to code coverage, and adding an extra step to build in case it's jest tests